src/components/shared/liskAmount/index.js

```diff
@@ -10,11 +10,19 @@ const roundToPlaces = (value, places) => {
   return Math.round(value * x) / x;
 };
 
-const LiskAmount = ({ val, roundTo }) => (
+const LiskAmount = ({ val, roundTo, token }) => (
```

Comment:
Because this component renders the amount based on different tokens, it doesn't make scence to be called `liskAmount` anymore.

Reply:
Totally agree. Should I fix it in this PR @yasharAyari ? 

There are many places that need to be updated:
```sh
$ git grep LiskAmount
src/components/screens/delegates/delegatesListView/delegateRow.js:import LiskAmount from '../../../shared/liskAmount';
src/components/screens/delegates/delegatesListView/delegateRow.js:          <LiskAmount val={rewards} />
src/components/screens/delegates/delegatesListView/delegateRow.js:          <LiskAmount val={data.vote} />
src/components/screens/explorer/transactions/transactions.js:import LiskAmount from '../../../shared/liskAmoun:...skipping...
src/components/screens/delegates/delegatesListView/delegateRow.js:import LiskAmount from '../../../shared/liskAmount';
src/components/screens/delegates/delegatesListView/delegateRow.js:          <LiskAmount val={rewards} />
src/components/screens/delegates/delegatesListView/delegateRow.js:          <LiskAmount val={data.vote} />
src/components/screens/explorer/transactions/transactions.js:import LiskAmount from '../../../shared/liskAmount';
src/components/screens/explorer/transactions/transactions.js:                            <LiskAmount val={transaction.amount} />
src/components/screens/explorer/transactions/transactions.js:                      <LiskAmount val={transaction.fee} />
src/components/screens/hwWalletLogin/selectAccount/accountCard.js:import LiskAmount from '../../../shared/liskAmount';
src/components/screens/hwWalletLogin/selectAccount/accountCard.js:          <LiskAmount val={account.balance} />
src/components/screens/monitor/blockDetails/blockDetails.js:import LiskAmount from '../../../shared/liskAmount';
src/components/screens/monitor/blockDetails/blockDetails.js:                      <LiskAmount val={blockDetails.data.reward} token={t('LSK')} />
src/components/screens/monitor/blockDetails/blockDetails.js:                      <LiskAmount val={blockDetails.data.totalFee} token={t('LSK')} />
src/components/screens/monitor/blockDetails/blockDetails.js:                      <LiskAmount val={blockDetails.data.totalForged} token={t('LSK')} />
src/components/screens/monitor/blockDetails/blockDetails.js:                      <LiskAmount val={blockDetails.data.totalAmount} token={t('LSK')} />
src/components/screens/monitor/blocks/blocks.js:import LiskAmount from '../../../shared/liskAmount';
src/components/screens/monitor/blocks/blocks.js:    getValue: block => <LiskAmount val={block.totalAmount} token="LSK" />,
src/components/screens/monitor/blocks/blocks.js:    getValue: block => <LiskAmount val={block.totalForged} token="LSK" />,
src/components/screens/wallet/transactions/amount/TransactionAmount.js:import LiskAmount from '../../../../shared/liskAmount';
src/components/screens/wallet/transactions/amount/TransactionAmount.js:              <LiskAmount val={transaction.amount} roundTo={roundTo} />
src/components/screens/wallet/transactions/transactionRow.js:import LiskAmount from '../../../shared/liskAmount';
src/components/screens/wallet/transactions/transactionRow.js:          <LiskAmount val={value.fee} token={token} />
src/components/screens/wallet/walletDetails.js:import LiskAmount from '../../shared/liskAmount';
src/components/screens/wallet/walletDetails.js:                <LiskAmount val={balance} />
src/components/shared/delegate/delegateTab.js:import LiskAmount from '../liskAmount';
src/components/shared/delegate/delegateTab.js:              <LiskAmount val={delegate.vote} token={t('LSK')} />
src/components/shared/delegate/delegateTab.js:              <LiskAmount val={delegate.rewards} />
src/components/shared/header/topBar/accountMenu/accountInfo.js:import LiskAmount from '../../../liskAmount';
src/components/shared/header/topBar/accountMenu/accountInfo.js:          <LiskAmount val={account.balance} />
src/components/shared/liskAmount/index.js:const LiskAmount = ({ val, roundTo, token }) => (
src/components/shared/liskAmount/index.js:LiskAmount.defaultProps = {
src/components/shared/liskAmount/index.js:export default LiskAmount;
src/components/shared/liskAmount/index.test.js:import LiskAmount from '.';
src/components/shared/liskAmount/index.test.js:describe('LiskAmount', () => {
src/components/shared/liskAmount/index.test.js:    const wrapper = mount(<LiskAmount val={inputValue} />, {
src/components/shared/liskAmount/index.test.js:    const wrapper = mount(<LiskAmount val={inputValue} roundTo={2} />, {
src/components/shared/searchBar/transactions.js:import LiskAmount from '../liskAmount';
src/components/shared/searchBar/transactions.js:                <LiskAmount val={transactionType.value} />
src/components/shared/votes/votesTab.js:import LiskAmount from '../liskAmount';
src/components/shared/votes/votesTab.js:                        <LiskAmount val={vote.rewards} />
src/components/shared/votes/votesTab.js:                        <LiskAmount val={vote.vote} />
src/components/shared/walletDetails/walletDetails.js:import LiskAmount from '../liskAmount';
src/components/shared/walletDetails/walletDetails.js:                  <LiskAmount val={coin.balance} />
```

Reply:
I created a follow-up ticket for this: https://github.com/LiskHQ/lisk-hub/issues/2535